### ISSUE TITLE
google maps api in contact.php

### ISSUE THREAD

**Issue by SwimGeek** [Obox/layerswp#328]
Hi

Seems Layers generates this in contact.php

script type='text/javascript' src='//maps.googleapis.com/maps/api/js?sensor=false&#038;ver=4.4.2'></script>

it gives this error:

"Google Maps API warning: SensorNotRequired https://developers.google.com/maps/documentation/javascript/error-messages#sensor-not-required"

It seems to add this line even if I don't use the Contact Details and Map Widget.

It caused my Gravity Form Geo plugin to break because the API loads twice.

**Comment by vailjoy** [comment]
Fixed in 1.2.14

**Comment by SwimGeek** [comment]
I upgraded to 1.2.14 but I still see an error:
...

You have included the Google Maps API multiple times on this page. This may cause unexpected errors. js:100:295

"Google Maps API warning: NoApiKeys https://developers.google.com/maps/documentation/javascript/error-messages#no-api-keys"

....

I attached a version of the file where it's been fixed.
[contact.php.txt](https://github.com/Obox/layerswp/files/193563/contact.php.txt)
